lib/api/controllers/funnelController.js

```diff
@@ -200,16 +201,19 @@ class FunnelController {
       return -1;
     }
 
-    let result = 0;
+    // if the connection is closed there is no need to execute the request
+    // => discarding it
+    if (!this.kuzzle.router.isConnectionAlive(request.context)) {
+      delete this.pendingRequests[request.id];
+      return 0;
+    }
+
     this.checkRights(request)
       .then(modifiedRequest => this.processRequest(modifiedRequest))
       .then(processResult => callback(null, processResult))
-      .catch(err => {
-        this._executeError(err, request, true, callback);
-        result = 1;
-      });
+      .catch(err => this._executeError(err, request, true, callback));
```

Comment:
OK, after reading more carefully the code, its clearer for me...
Just the jsdoc line `0: ok, 1: error, -1: delayed` is quite confusing with what kind of "errors" we are speaking at.

maybe we should just me more explicit in the jsdoc, with something like 
```
  * @returns {Number} -1: request delayed, 0: request processing, 1: error while trying to get the request slot
```
?